etc/services/preprints-lissa.json

```diff
@@ -1,7 +1,7 @@
 {
   "@class" : "org.jasig.cas.services.RegexRegisteredService",
   "id" : 203948234207249,
-  "name" : "LISSA",
+  "name" : "LIS Scholarship Archive Preprints",
```

Comment:
Use the same name on OSF and Preprints. "LISSA Preprints" becomes "LIS Scholarship Archive Preprints".